src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1018,16 +1006,14 @@ void BlockchainLMDB::remove_transaction_data(const crypto::hash& tx_hash, const
         throw1(DB_ERROR(lmdb_error("Error adding removal of tx id to db transaction", result).c_str()));
   }
 
-  if (tx.version > 1)
-  {
-    if ((result = mdb_cursor_get(m_cur_txs_prunable_hash, &val_tx_id, NULL, MDB_SET)))
-        throw1(DB_ERROR(lmdb_error("Failed to locate prunable hash tx for removal: ", result).c_str()));
-    result = mdb_cursor_del(m_cur_txs_prunable_hash, 0);
-    if (result)
-        throw1(DB_ERROR(lmdb_error("Failed to add removal of prunable hash tx to db transaction: ", result).c_str()));
-  }
-
-  remove_tx_outputs(tip->data.tx_id, tx);
+  // Remove prunable hash
+  result = mdb_cursor_get(m_cur_txs_prunable_hash, &val_tx_id, NULL, MDB_SET);
+  if (result == MDB_NOTFOUND)
+    LOG_PRINT_L1("tx has no prunable hash to remove: " << tx_hash);
+  else if (result)
+    throw1(DB_ERROR(lmdb_error("Failed to locate prunable hash tx for removal: ", result).c_str()));
+  else if ((result = mdb_cursor_del(m_cur_txs_prunable_hash, 0)))
+      throw1(DB_ERROR(lmdb_error("Failed to add removal of prunable hash tx to db transaction: ", result).c_str()));
```

Comment:
apologies for nitpicking. fix the indentation here to have a consistent indentation with other `throw1`s.